### PR TITLE
Testing views 104

### LINKED ISSUE #104: Add information about testing views
How Views can be tested: https://discuss.lightbend.com/t/how-to-test-views/8427/2

### PR DESCRIPTION
**Pull request by rstento**
Fixes #104

### CODE REVIEW

File: docs/src/modules/javascript/pages/views.adoc [hunk 1]
```diff
@@ -213,6 +214,37 @@ include::example$js-customer-registry/customer_view.proto[tag=wrap-repeated]
...
 <3> The `repeated` field is referenced in the query with `* AS results`.
 
+[#testing]
+== How to test a View
+
+View tests need to create gRPC clients for both the Entity and the View. For example:
+
+[source,javascript]
+----
+include::example$js-customer-registry/integration-test/customer-registry-test.js[lines=17..30]
```
Comment (pvlugter) [inline docs/src/modules/javascript/pages/views.adoc:224]: Would be much better to use tags in the source files, rather than line numbers. We could update the test and never realise it's referenced from the docs.

File: samples/js-customer-registry/customer_domain.proto [hunk 2]
```diff
@@ -12,9 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// tag::declarations[]
```
Comment (pvlugter) [inline samples/js-customer-registry/customer_domain.proto:15]: Add these tags seem unrelated and unused in this PR.

**Review by pvlugter — COMMENTED** [review]
Looks fine, but would be good to use tags for code snippets, not line numbers.

File: samples/js-customer-registry/customer_domain.proto (hunk 2 above)
Comment (rstento) [reply samples/js-customer-registry/customer_domain.proto:15]: oops, slipped in from my other PR. Removed.

**Review by pvlugter — APPROVED** [review]
Thanks 👍🏼
